docs/source/conf.py

```diff
@@ -31,6 +31,7 @@
 #
 # needs_sphinx = '1.0'
 
+
```

Comment:
Sorry to knitpick, but if you make any other changes please rm this whitespace change.

Reply:
Ah, sorry I missed that!